### DISCUSSION TITLE
I don't read iOS app store release notes

### DISCUSSION THREAD

**Discussion by verhovsky** [organicmaps/organicmaps#9039] [Q&A]
I just saw this paradoxical question in the App Store,

![image](https://github.com/user-attachments/assets/8225ab20-f7d4-4dff-87c1-4834f204969c)

and this is my answer. No I don't and I've only done it a few times accidentally because they're right there when you go to manually update your iPhone apps. Most apps have a generic empty statement here like "fixed bugs" for every single release.

![image](https://github.com/user-attachments/assets/8cda6c14-bfa5-495d-bf70-431dcd6a9974)

**Comment by jkhsjdhjs** [comment]
Also came here to comment on this. Yes, I read the changelog and I hate that some developers only put trash in there (especially Google).

**Reply by verhovsky** [reply]
This question is textbook selection bias.

**Reply by polyzen** [reply]
I too read release notes.

**Reply by gnojus** [reply]
Agreed. As someone who has auto-update turned off and updates apps manually, it's always a pleasure to read how Organic Maps, already a great app, has improved.

**Reply by polyzen** [reply]
To be fair, I literally install updates from GitHub and read the release notes here 😅 (and/or my emails from tracking the GH release feed). That being said, I always do appreciate proper release notes for apps I need to install from an app store.

**Reply by eisams** [reply]
Yup, I also read the release notes

**Comment by not-available-username** [comment]
I always read the release notes!
